Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.cpp

```diff
@@ -790,17 +791,24 @@ namespace AzFramework
                 const auto* position = inputChannel.GetCustomData<AzFramework::InputChannel::PositionData2D>();
                 AZ_Assert(position, "Expected PositionData2D but found nullptr");
 
-                return CursorEvent{ ScreenPoint(
-                    static_cast<int>(position->m_normalizedPosition.GetX() * windowSize.m_width),
-                    static_cast<int>(position->m_normalizedPosition.GetY() * windowSize.m_height)) };
+                auto currentCursorState = AzFramework::SystemCursorState::Unknown;
```

Comment:
I was hoping to avoid having to bubble the concept of cursor capture around too much, but this looks like a sensible way to handle the context menu issue.

Reply:
Thanks for the context Nick, I tried for a while to see if it was possible to avoid but it just kept being more complicated to try and workaround it, fortunately it slots in fairly painlessly and might prove useful in future.